src/mono/mono/metadata/debug-mono-ppdb.c

```diff
@@ -30,12 +30,8 @@
 #include <mono/utils/mono-logger-internals.h>
 
 #ifndef DISABLE_EMBEDDED_PDB
-#ifdef INTERNAL_ZLIB
-#include <external/zlib/zlib.h>
-#else
 #include <zlib.h>
```

Comment:
It's my understanding that since we're enabling ZLIB_COMPAT in zlib-ng, we should be able to continue consuming <zlib.h>.